src/main/resources/reference.conf

```diff
@@ -1384,6 +1381,14 @@ literature {
         format = ${common.output-format}
         path = ${literature.output}"/literatureSentences"
       }
+      failed-cooccurrences {
+        format = ${common.output-format}
+        path = ${literature.output}"/cooccurrences"
+      }
+      failed-matches {
+        format = ${common.output-format}
+        path = ${literature.output}"/matches"
```

Comment:
This is the same case, you can see at opentargets/issues#2993 how this data was being dumped in a folder with name _failedMatches_, not _matches_
So the new destination should be
```
${literature.output}"/failedMatches"
```